### DISCUSSION TITLE
Augmented Reality for Architecture (Project 240)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#76] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 240](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Augmented%20Reality%20for%20Architecture)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by FidelisWandia** [comment]
Hello. My name is Wandia. I am interested in working and collaborating on this project.

**Reply by Hari-MVS** [reply]
Thank you for reaching out to me about this project. I'm interested in collaborating with you.

**Comment by FidelisWandia** [comment]
Hello Hari. Thank you for reaching out. Looking forward to chatting more
about this. What is your preferred means of communication?

**Reply by JrMeela** [reply]
Hello!. Is anyone in here still working on this Project?
